conf/clusters/prod.json

```diff
@@ -31,8 +31,8 @@
         "current_version": "$LATEST",
         "enable_metrics": true,
         "log_level": "info",
-        "memory": 128,
-        "timeout": 10
+        "memory": 512,
```

Comment:
Can we make this configurable, depending on whether threat intel is enabled, since you pay for this on every invocation? See: https://aws.amazon.com/lambda/pricing/

Reply:
You bring up a good point. This parameter is defined in the json format and used by terraform. We can not add logic here. Since Threat Intel feature is disabled by default, I will change `memory` size back to 128.